js/core/renderer.d.ts

```diff
@@ -0,0 +1,256 @@
+interface JQueryCoordinates {
+    left: number;
+    top: number;
+}
+
+interface JQueryAnimationOptions {
+
+    duration?: any;
+
+    easing?: string;
+
+    complete?: Function;
+
+    step?: (now: number, tween: any) => any;
+
+    progress?: (animation: JQueryPromise<any>, progress: number, remainingMs: number) => any;
+
+    start?: (animation: JQueryPromise<any>) => any;
+
+    done?: (animation: JQueryPromise<any>, jumpedToEnd: boolean) => any;
+
+    fail?: (animation: JQueryPromise<any>, jumpedToEnd: boolean) => any;
+
+    always?: (animation: JQueryPromise<any>, jumpedToEnd: boolean) => any;
+
+    queue?: any;
+
+    specialEasing?: Object;
+}
+
+export interface dxElementWrapper {
+    
+    addClass(className: string): JQuery;
+    
+    attr(attributeName: string, value: string|number|null): JQuery;
+    
+    attr(attributeName: string, func: (index: number, attr: string) => string|number): JQuery;
+    
+    hasClass(className: string): boolean;
+    
+    html(htmlString: string): JQuery;
+    
+    html(func: (index: number, oldhtml: string) => string): JQuery;
```

Comment:
It seems that this overload is not implemented:

```js
initRender.prototype.html = function(value) {
    if(!arguments.length) {
        return this[0].innerHTML;
    }

    this.empty();

    if(typeof value === 'string' && !isTablePart(value) || typeof value === 'number') {
        this[0].innerHTML = value;

        return this;
    }

    return this.append(parseHTML(value));
};
```

Reply:
I think we should check for unsupported declarations and remove them.
And see if it breaks compilation 🍿

Reply:
Done